### DISCUSSION TITLE
Deploy verdaccio in kubernetes

### DISCUSSION THREAD

**Discussion by algarciatech** [verdaccio/verdaccio#2185] [Q&A]
Good afternoon, I'm trying to deploy them in AKS to be able to upload our packages. The fact is that everything is fine except that I cannot apply my config.yaml configuration since this is not generated when executing the deployment.yaml. file directly in the pod but it does not take the configuration and continues using the default.

I have tried the following:

-Load the config.yaml file directly with a configMap and with a Persistent Volume.
-Use a commit image of my local reality in docker
-Deploy with helm

All this has resulted in the pod starting but without the configuration that I want.

**Comment by juanpicado** [comment]
Maybe you can ask here https://github.com/verdaccio/charts @verdaccio/kubernetes are the experts :) that might help you out.
